src/main/resources/upgrade-files/4.0_4.1/db/master-structure-script_mod.xml

```diff
@@ -21,7 +21,9 @@
     <!--
 	<include file="01_structure/kc-integration-structure.xml"/>
         -->
+<!-- Lingering change from UAF-2544. Discovered while testing UAF-2859
```

Comment:
Good catch, but db/master-structure-script_mod.xml is not in the files-4.0_4.1 list of upgrade files to be executed. Shouldn't have caused an issue.